src/Nethermind/Nethermind.Network.Stats/SyncLimits/BeSuSyncLimits.cs

```diff
@@ -1,12 +1,12 @@
 // SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
 // SPDX-License-Identifier: LGPL-3.0-only
 
-namespace Nethermind.Synchronization.SyncLimits
+namespace Nethermind.Stats.SyncLimits
 {
     public static class BeSuSyncLimits
     {
         public const int MaxHeaderFetch = GethSyncLimits.MaxHeaderFetch; // Amount of block headers to be fetched per retrieval request
```

Comment:
Have you double check on besu codebase? Can you link the specific part of the code here?